scripts/linters/pylint_extensions.py

```diff
@@ -1893,6 +1893,42 @@ def visit_compare(self, node):
                 self.add_message('inequality-with-none', node=node)
 
 
+class NonTestFilesFunctionNameChecker(checkers.BaseChecker):
+    """Custom pylint checker prohibiting use of "test_only" in function names
+    of non-test files.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+
+    name = 'non-test-files-function-name-checker'
+    priority = -1
+    msgs = {
+        'C0031': (
+            'Please refrain from using test_only in function names '
+            'in case of non-test files.',
+            'non-test-files-function-name-checker',
+            'Try using a different name for the function that does not have'
+            'test_only as its prefix.'
```

Comment:
Rephrase the 1st and 3rd message such that the 1st message should explain what a user needs to do to fix the issue in the codebase. and the 3rd message should be info about this lint check i.e, what this lint check provides?

Reply:
Done.